packages/test/test-service-load/src/runner.ts

```diff
@@ -224,6 +224,9 @@ async function runnerProcess(
 			container.connect();
 			const test = await requestFluidObject<ILoadTest>(container, "/");
 
+			// Retain old behavior of runtime being disposed on container close
+			container.once("closed", () => container?.dispose());
```

Comment:
can you revert the other changes of .close() to .dispose(), file an issue, and link it here about fixing stress to work with the new model of close and dispose.

Reply:
the one dispose in utils is fine to keep as well.

Reply:
[AB#4137](https://dev.azure.com/fluidframework/internal/_workitems/edit/4137)